src/AppInstallerRepositoryCore/Microsoft/SQLiteIndex.h

```diff
@@ -51,6 +51,10 @@ namespace AppInstaller::Repository::Microsoft
         // Gets the schema version of the index.
         Schema::Version GetVersion() const { return m_version; }
 
+        // Changes the version of the interface being used to operate on the database.
+        // Should only be used for testing.
+        void ForceVersion(const Schema::Version& version);

```

Comment:
>void ForceVersion(const Schema::Version& version); [](start = 8, length = 50)

should we put it inside existing #def DISABLE***TEST_HOOK since it's test only?